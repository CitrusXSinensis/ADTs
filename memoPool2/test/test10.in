ra1 2x
pa
ca
ra0x
ra1x
ca
ra22 3 2 4 6 2x
ca
pa

