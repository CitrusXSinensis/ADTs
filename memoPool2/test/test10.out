1 2
2/4
1/4
6/8
22 3 2 4 6 2
